jgrapht-demo/src/main/java/org/jgrapht/demo/GraphMLExportDemo.java

```diff
@@ -0,0 +1,260 @@
+/* ==========================================
+ * JGraphT : a free Java graph-theory library
+ * ==========================================
+ *
+ * Project Info:  http://jgrapht.sourceforge.net/
+ * Project Creator:  Barak Naveh (http://sourceforge.net/users/barak_naveh)
+ *
+ * (C) Copyright 2003-2008, by Barak Naveh and Contributors.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+/* --------------------
+ * GraphMLExportDemo.java
+ * --------------------
+ * (C) Copyright 2016-2016, by Dimitrios Michail and Contributors.
+ *
+ * Original Author:  Dimitrios Michail
+ * Contributor(s):   -
+ *
+ * Changes
+ * -------
+ * 26-July-2016 : Initial revision (DM); 
+ *
+ */
+package org.jgrapht.demo;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import org.jgrapht.VertexFactory;
+import org.jgrapht.ext.ComponentAttributeProvider;
+import org.jgrapht.ext.ExportException;
+import org.jgrapht.ext.GraphMLExporter;
+import org.jgrapht.ext.VertexNameProvider;
+import org.jgrapht.ext.GraphMLExporter.AttributeCategory;
+import org.jgrapht.ext.GraphMLExporter.AttributeType;
+import org.jgrapht.ext.IntegerEdgeNameProvider;
+import org.jgrapht.generate.CompleteGraphGenerator;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.DirectedPseudograph;
+
```

Comment:
Maybe add a short description about what the demo exactly does, and how it achieves this. E.g "This class demonstrates the GraphMLExmporter by .... "